lib/src/currencies.dart

```diff
@@ -1095,7 +1095,7 @@ List<Map<String, dynamic>> currencies = [
   {
     "code": "TTD",
     "name": "Trinidad and Tobago Dollar",
-    "symbol": "TT\$",
+    "symbol": "TT$\$",
```

Comment:
"TT$\$" is not valid string in dart. For symbol TT$ string string be "TT\$".

Reply:
ohhh, my bad. I didn't know that.

Reply:
```suggestion
    "symbol": "TT\$",
```

Reply:
@Lakshan-Costa Please fix it so I can approve your PR

Reply:
@Daniel-Ioannou I'm new to github, was "commit suggestion" you were looking for when you said "please fix it"?